pkg/log/syslog/mock_netDialer_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2016-2017 VMware, Inc. All Rights Reserved.
```

Comment:
Should this filename be camel-cased like that? Or should it be `mock_netdialer_test.go`?
Same with `mock_netConn_test.go` vs. `mock_netconn_test.go`.